<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="qapitol-ThinkPad-L470" ignored="0" name="Test" tests="5" failures="1" timestamp="2019-09-16T23:48:45 IST" time="9.08" errors="0">
  <testcase name="ValidateResponse" time="0.084" classname="com.restAssuredEmployeeApi.testCases.AddEmployee"/>
  <testcase name="ValidateResponse" time="0.402" classname="com.restAssuredEmployeeApi.testCases.GetAllEmployees"/>
  <testcase name="ValidateResponse" time="0.002" classname="com.restAssuredEmployeeApi.testCases.DeleteEmployee">
    <failure type="java.lang.AssertionError" message="Employee is not Deleted did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Employee is not Deleted did not expect to find [true] but found [false]
at com.restAssuredEmployeeApi.testCases.DeleteEmployee.ValidateResponse(DeleteEmployee.java:60)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ValidateResponse -->
  <testcase name="ValidateResponse" time="0.042" classname="com.restAssuredEmployeeApi.testCases.UpdateEmployee"/>
  <testcase name="ValidateResponse" time="0.013" classname="com.restAssuredEmployeeApi.testCases.GetEmployee"/>
</testsuite> <!-- Test -->
